mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule PhoenixTokenAuth.Mixfile do
   def project do
     [app: :phoenix_token_auth,
      version: "0.0.14",
-     elixir: "~> 1.0",
+     elixir: "~> 1.0.4",
```

Comment:
This breaks the travis-ci build. Is there a reason we can't support < 1.0.4 anymore?